matlab/lib/+Ice/Communicator.m

```diff
@@ -217,7 +217,7 @@ function destroy(obj)
             impl = libpointer('voidPtr');
             obj.iceCall('getDefaultRouter', impl);
             if ~isNull(impl)
-                r = Ice.RouterPrx(impl, obj);
```

Comment:
This was totally bogus, so I suspect this code is not tested. Same for `getDefaultLocator` below.